cpp/src/fil/treelite_import.cu

```diff
@@ -655,17 +655,23 @@ void convert(const raft::handle_t& handle,
   tl2fil.init_forest(handle, pforest);
 }
 
+template <typename real_t>
+constexpr bool type_supported()
+{
+  // not using std::is_floating_point because we did not instantiate fp16-based nodes/trees/forests
+  return std::is_same<real_t, float>() || std::is_same<real_t, double>();
```

Comment:
This is not a required change, but personally I find `std::is_same_v` more readable than an implicit conversion to `operator bool`

Reply:
Done.